### DISCUSSION TITLE
Is there a sensor for asset materialization failures?

### DISCUSSION THREAD

**Discussion by achen187** [dagster-io/dagster#17574] [Q&A]
I want to do some logic if an asset materialization fails. Sorta the equivalent of this, but for assets and not jobs: https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors#run-failure-sensor

I looked at asset_sensors, but it seems like those only work when the materialization succeeds?

**Comment by wonjae-2352** [comment]
@achen187 would you mind sharing if you find any good way to set sensor on asset failure?
